content/pandas/pandas.md

```diff
@@ -4,6 +4,7 @@ Description: 'Pandas is a popular open-source Python module, used for data analy
 Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/python'
 CatalogContent:
   - 'learn-python-3'
+  - 'getting-started-with-python-for-data-science'
   - 'paths/data-science'
```

Comment:
```suggestion
  - 'getting-started-with-python-for-data-science'
  - 'paths/data-science'
```

I agree that this new link is more relevant to pandas as a topic. Let's keep this to two slugs, though. (one free slug and one paid course slug)